app/views/layouts/admin/_notifications.html.erb

```diff
@@ -0,0 +1,40 @@
+<a class="nav-link dropdown-toggle" 
+   href="#" id="alertsDropdown"
+   role="button"
+   data-toggle="dropdown"
+   aria-haspopup="true"
+   aria-expanded="false">
+  <i class="fas fa-bell fa-fw text-light"></i>
+  <span class="badge badge-danger badge-counter">3+</span>
+</a>
+<div class="dropdown-list dropdown-menu dropdown-menu-right shadow animated--grow-in"
+     aria-labelledby="alertsDropdown">
+  <h6 class="dropdown-header bg-dark border-dark">
+    Alerts Center
+  </h6>
+  <a class="dropdown-item d-flex align-items-center" href="#">
```

Comment:
the notification dropdown should use the notification levels (info, warning, critial, priority).

![image](https://user-images.githubusercontent.com/8007722/72945752-cb8b4e00-3d4a-11ea-984b-d54652e74483.png)